tensorflow/tools/pip_package/build_pip_package.sh

```diff
@@ -41,46 +51,67 @@ function main() {
     exit 1
   fi
 
-  if [ ! -d bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles/org_tensorflow ]; then
-    # Really old (0.2.1-) runfiles, without workspace name.
+  if is_windows; then
+    mkdir -p ./bazel-bin/tensorflow/tools/pip_package/simple_console_for_window_unzip
+    echo "Unzipping simple_console_for_windows.zip to create runfiles tree..."
+    unzip -o -q ./bazel-bin/tensorflow/tools/pip_package/simple_console_for_windows.zip -d ./bazel-bin/tensorflow/tools/pip_package/simple_console_for_window_unzip
+    echo "Unzip finished."
+    # runfiles structure after unzip the python binary
     cp -R \
-      bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles/tensorflow \
+      bazel-bin/tensorflow/tools/pip_package/simple_console_for_window_unzip/runfiles/org_tensorflow/tensorflow \
       "${TMPDIR}"
     mkdir "${TMPDIR}/external"
+    # Note: this makes an extra copy of org_tensorflow.
     cp_external \
-      bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles/external \
+      bazel-bin/tensorflow/tools/pip_package/simple_console_for_window_unzip/runfiles \
       "${TMPDIR}/external"
-    RUNFILES=bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles
+    RUNFILES=bazel-bin/tensorflow/tools/pip_package/simple_console_for_window_unzip/runfiles/org_tensorflow
   else
-    if [ -d bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles/org_tensorflow/external ]; then
-      # Old-style runfiles structure (--legacy_external_runfiles).
+    if [ ! -d bazel-bin/tensorflow/tools/pip_package/build_pip_package.runfiles/org_tensorflow ]; then
```

Comment:
Nit: Can you make this an `elif`? It's difficult to see what code has changed between the old and new version, so `elif` would let you have the same logic without changing so many lines.